Userland/Libraries/LibGfx/Emoji.h

```diff
@@ -8,6 +8,7 @@
 #pragma once
 
 #include <AK/Types.h>
+#include <AK/Utf8View.h>
```

Comment:
Extreme nitpick but perhaps `Utf8CodePointIterator` should be forward declared? Then we can avoid a number of these includes in the header? (Definitely not needed)

Reply:
Sure, done :+1: